src/algebra/linear-diophantine-equation.md

```diff
@@ -51,6 +51,12 @@ y = \frac{c-ax}{b}.
 
 ## Algorithmic solution
 
+**Bézout's lemma** (also called Bézout's identity) is a useful result that can be used to understand and prove the following solution. 
```

Comment:
I think it's actually the other way around: the solution is an algorithmic proof to Bezout's lemma, rather than the lemma is a proof for the solution.

Reply:
It can prove the existence part. I can reword it